### ISSUE TITLE
BatchNorm initialization

### ISSUE THREAD

**Issue by ChengshuLi** [CSAILVision/semantic-segmentation-pytorch#100]
Hi, 

Quick question for BatchNorm initialization.
https://github.com/CSAILVision/semantic-segmentation-pytorch/blob/24e13b30f1af9f8ae7fb80b5cace899be3d1794a/models/models.py#L68-L70

Is there a particular reason you want to initialize BatchNorm this way instead of the default way? Is it any theory behind this or is it purely empirical?

Thanks a lot!

**Comment by ChengshuLi** [comment]
@hangzhaomit
@Tete-Xiao

**Comment by hangzhaomit** [comment]
@ChengshuLi It is empirical, or a common practice.
